packages/apps/spaces/app/organizations/[id]/components/Timeline/events/meeting/MeetingPreviewModal.tsx

```diff
@@ -0,0 +1,339 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import { convert } from 'html-to-text';
+import { useForm } from 'react-inverted-form';
+import { utcToZonedTime, format } from 'date-fns-tz';
```

Comment:
Move those to DateUtils class, it's to keep those fuctions after fasade to ensure easiness on changing library if we'll ever want to do that for some reason